clang/test/CodeGen/attr-cpuspecific.c

```diff
@@ -114,8 +114,8 @@ void ThreeVersionsSameAttr(void){}
 // CHECK: define {{.*}}void @ThreeVersionsSameAttr.Z() #[[K]]
 
 ATTR(cpu_specific(knl))
-void CpuSpecificNoDispatch(void) {}
-// CHECK: define {{.*}}void @CpuSpecificNoDispatch.Z() #[[K:[0-9]+]]
+void CpuSpecificNoDispatch(void (*f)(void)) {}
+// CHECK: define {{.*}}void @CpuSpecificNoDispatch.Z(ptr noundef %f) #[[K:[0-9]+]]
```

Comment:
Why the change in the signatures here?  We should leave the old version of the test around, but if you want a pointer argument test, just add it.

Reply:
The change is to trigger the crash. I am not sure why we are not crashing without an argument. I can add a new test then and keep this too.